scipy/stats/stats.py

```diff
@@ -6694,8 +6713,8 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
            Statistics". Chapter 8.
            https://web.archive.org/web/20171022032306/http://vassarstats.net:80/textbook/ch8pt1.html
     .. [2] "Chi-squared test", https://en.wikipedia.org/wiki/Chi-squared_test
-    .. [3] Pearson, Karl. "On the criterion that a given system of deviations from the probable 
-           in the case of a correlated system of variables is such that it can be reasonably 
+    .. [3] Pearson, Karl. "On the criterion that a given system of deviations from the probable
+           in the case of a correlated system of variables is such that it can be reasonably
```

Comment:
Same here:

```suggestion
    .. [3] Pearson, Karl. "On the criterion that a given system of deviations from the probable 
           in the case of a correlated system of variables is such that it can be reasonably 
```